modules/jupyter-widget/src/create-deck.js

```diff
@@ -96,6 +123,16 @@ function createDeck({
       container
     });
 
+    const onComplete = () => {
+      const newProps = jsonConverter.convert({layers: layerToLoad});
+      const mergeLayers = [...props.layers, ...newProps.layers].filter(l => l);
```

Comment:
- Add some comments on why we need to do this
- Doesn't this change the order of layers?